src/views/Login.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function Login () {
+  return (
+    <div />
```

Comment:
You should add text to make this more clear for future devs and people while in transition state. Simply do `<div>Login</div>`.